besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -3073,6 +3089,15 @@ public void onchainPrivacyAndMultiTenancyCannotBeUsedTogether() {
         .startsWith("Privacy multi-tenancy and onchain privacy groups cannot be used together");
   }
 
+  @Test
+  public void privacyMarkerTransactionSigningKeyFileRequiredIfMinGasPriceNonZero() {
+    parseCommand("--privacy-enabled", "--privacy-public-key-file", ENCLAVE_PUBLIC_KEY_PATH);
```

Comment:
Does it make sense to rely on a default non-zero gas price, or would it be clearer to have an explicit non-zero gas price?

Reply:
changing this is outside the scope of this PR